service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellService.java

```diff
@@ -123,4 +123,8 @@ public static String getErrorMessage(List<FailureMessage> failureMessages) {
       return result;
     }
   }
+
+  public RunId submitAbortWorkflow(Run run) throws ApiException {
```

Comment:
This naming feels a little obtuse... is there a reason to not also call this "cancelRun"?